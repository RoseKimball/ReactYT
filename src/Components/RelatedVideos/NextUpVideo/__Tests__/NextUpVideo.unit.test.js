import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, mount, render } from 'enzyme';
import NextUpVideo from '../NextUpVideo';

describe('NextUpVideo', () => {
    test('renders NextUpVideo', () => {
        const wrapper = shallow (
            <NextUpVideo />
        )
        expect(wrapper).toMatchSnapshot();
    })
})