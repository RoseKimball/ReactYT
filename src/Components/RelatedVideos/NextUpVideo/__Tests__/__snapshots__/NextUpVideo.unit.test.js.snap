// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NextUpVideo renders NextUpVideo 1`] = `ShallowWrapper {}`;
